CONTRIBUTING.md

```diff
@@ -23,7 +23,7 @@ The recommended way of setting your environment up is with [Anaconda](https://ww
 
 The project uses Python 3 (specified in the `python=3` bit of the commands below). [Download Anaconda](https://www.continuum.io/downloads) and create an _environment_ for the project.
 
-The version of Anaconda doesn't have to be 3 as `conda` environments manage Python version as well. But if you're going to a fresh install we recommend Anaconda 3 — it's newer and it makes no sense to have Python 2 as your default Anaconda Python.
```

Comment:
Can you rollback this line, please? I mean… your version is way less precise what what we had:

* Both Anaconda and Anaconda3 can manage Python 2 and Python 3 environment under the hood, so I prefer _the version of Anaconda doesn't have to be 3_ instead of _the Python version in Anaconda doesn't have to be 3_;
* `coda` environments can use different Python versions, so I prefer _`conda` environments manage Python version as well_ instead of _`conda` environments manage other versions as well_);
* Actually we recommend Anaconda3 (and Python 3 is managed by the `conda` environment, so I prefer _we recommend Anaconda 3_ instead of _we recommend Python 3_.